OSD-2022.adoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: OSD-2022
+title: Open Source Day 2022
+permalink: /OSD-2022/
+---
+:page-liquid:
+
+
+Welcome Open Source Day participants! We're glad to have you join us for the hackathon.
+
+To get started, follow the steps below:
+
+. *Set up the necessary tools to get started.* We use Java 11, Maven, and GitHub for development. Follow the set up instructions for your operating system:
++
+++++
+{% include OSD-2022-logo-bullets.html %}
+++++
+
+. *Pick your https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Java[IDE].* We recommend https://www.jetbrains.com/idea/[IDEA] or https://www.eclipse.org/ide/[Eclipse], but feel free to use whatever you'd like.
+. *Start contributing!* Whether you're new to security, new to Java, or new to tools like Git or Maven, we have a bunch of issues that are perfect for getting started on our project, no prior experience is needed. Check out our list of (https://issues.redhat.com/issues/?filter=12364234)[good first issues]. Once you have selected an issue you'd like to work on, make sure it's not already assigned to someone else. To assign an issue to yourself, simply click on "Start Progress". This will automatically assign the issue to you. +
+Our (https://github.com/wildfly-security/wildfly-elytron/blob/1.x/CONTRIBUTING.md#contributing-guidelines)[contribution guide] has all the information you need on how to format your PRs.
+
+
+
+If you have any questions, feel free to ask us on Slack or on Zoom.
+
+To get more familiar with WildFly, our link:../getting-started-for-developers/[Getting Started] page has some resources to help you learn more about the project. Want to get started quickly? Check out these sections:
+    ** link:../getting-started-for-developers/#a-quick-introduction-to-the-project[WildFly Project Introductions]
+    ** link:../getting-started-for-developers/#getting-familiar-with-elytron[Getting familiar with Elytron]
+    ** link:../getting-started-for-developers/#getting-your-developer-environment-set-up[Building and testsuite comamnds]
```

Comment:
@cam-rod  Just a minor, these links are did not render well, I can see double asterisk before the last link

Reply:
Oh, forgot to remove some formatting when I moved that section